requirements.txt

```diff
@@ -3,3 +3,5 @@ pyyaml
 click
 toml
 requests
+Flask
```

Comment:
Flask no longer needed, but you do need to add jinja (or jinja2?) here.

Reply:
I don't think I need to add jinja because I don't create the html file here. I think Sera has a command in her pull request that calls another file that needs jinja.